### PR TITLE
Fix packages

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Seems to have the same problem as https://github.com/elifesciences/bus-sdk-python/pull/8.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,9 @@
     name='api-validator-python',
     version=elife_api_validator.__version__,
     description='Utility for accessing and validating eLife API JSON Schemas.',
-    packages=['elife_api_validator'],
+    packages=['elife_api_validator',
+              'elife_api_validator.schemas',
+              'elife_api_validator.validators'],
     package_data={
         'elife_api_validator/schemas': '*.json'
```
Comment (thewilkybarkid) [inline setup.py]: This appears not to work too, reading the docs the key should be the package name (`elife_api_validator.schemas`)?

Comment (thewilkybarkid) [reply setup.py]: https://github.com/elifesciences/profiles/pull/89 breaks on both. Changing to this PR sees the `packages` work, then it doesn't find the schemas.

Comment (seanwiseman) [reply setup.py]: I have switched to decaring non python files in a `MANIFEST.ini` like I do in `proofreader-python`.

Is now including the `json` files when I build it locally.

Comment (thewilkybarkid) [reply setup.py]: 👍 Tests now green here.

**Comment by thewilkybarkid** [comment]
:shipit:?

**Pull request merged**